components/Navbar.js

```diff
@@ -5,17 +5,92 @@ import Logo from '../public/logo.svg';
 import Button from '../components/form/Button';
 import Link from 'next/link';
 
+const style = {
+  container: `relative top-1/4 w-full text-left pl-16 mt-8`,
+  item: `text-3xl text-amcovad-white cursor-pointer  hover:amcovad-secondary-100`,
+  menu: {
+    isMenuOpen: `h-full w-full `,
+    close: `w-0 h-full`,
+    default: `overflow-x-hidden md:overflow-hidden transition-all duration-700 fixed z-10 top-0 left-0 bg-amcovad-primary-500`
+  }
+};
+
+const Menu = ({ children, isMenuOpen }) => {
+  return (
+    <div
+      className={`${style.menu.default} 
+       ${isMenuOpen ? style.menu.isMenuOpen : style.menu.close}`}
+    >
+      {children}
+    </div>
+  );
+};
+
+const MenuContainer = ({ children }) => {
+  return <div className={style.container}>{children}</div>;
+};
+
+const MenuItem = ({ href, children }) => {
+  return (
+    <div className="p-2">
+      <Link href={href} passHref>
+        <span className={style.item}>{children}</span>
+      </Link>
+    </div>
+  );
+};
+
 const Navbar = () => {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
 
+  const toggle = () => {
+    setIsMenuOpen((prevState) => !prevState);
+  };
+
   return (
     <header className="fixed z-10 outline-[5px] border-b  border-amcovad-primary-200 2xl:mx-auto w-full">
       <div className="px-4 bg-amcovad-secondary-100 py-2 mx-auto md:max-w-full md:px-20 lg:px-8 2xl:px-60">
         <div className="lg:pl-8  lg:pr-16 relative flex items-center justify-between">
-          <a href="#" aria-label="Company" title="Company" className="inline-flex items-center xl:pl-11">
-            <Image src={Logo} width="167" height="42" alt="logo" />
-          </a>
-          <ul className="flex items-center hidden lg:flex">
+          <Link href="/" passHref>
+            <div className="inline-flex items-center cursor-pointer xl:pl-11">
+              <Image src={Logo} width="160" height="36" alt="logo" />
+            </div>
+          </Link>
+          <ul className="items-center hidden space-x-8 lg:flex">
+            <li>
+              <Link
```

Comment:
We can add `menu` too to the data folder and loop through the values. Something like this:

```js
const menu = [{ 
    title: 'About us',
    url: '/about',
    },
    ...
]
```